src/Menu/Menu.js

```diff
@@ -224,7 +230,7 @@ class Menu extends React.Component {
 
     this.state = {
       focusIndex: props.disableAutoFocus ? -1 : selectedIndex >= 0 ? selectedIndex : 0,
-      isKeyboardFocused: props.initiallyKeyboardFocused,
+      isKeyboardFocused: props.keyboardFocused !== null ? props.keyboardFocused : props.initiallyKeyboardFocused,
```

Comment:
if we set prop `keyboardFocused` as false, `isKeyboardFocused` should be false, even if  `initiallyKeyboardFocused` is true